include/boost/url/detail/impl/any_query_iter.ipp

```diff
@@ -275,15 +264,53 @@ copy_impl(
     char*& dest,
     char const* end) noexcept
 {
-    static constexpr auto cs =
-        pchars + '/' + '?';
     dest += pct_encode(
-        dest, end, key, {}, cs);
+        dest, end, key, {}, query_chars);
+    if(value)
+    {
+        *dest++ = '=';
+        dest += pct_encode(
+            dest, end, *value, {}, query_chars);
+    }
+}
+
+//------------------------------------------------
+
+void
+plain_value_iter_base::
+measure_impl(
+    string_view key,
+    string_view const* value,
+    std::size_t& n) noexcept
+{
+    n += key.size();
```

Comment:
but.. is `key` encoded?

Reply:
Yes. This is just the key in the underlying URL. It doesn't come from outside.